tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary Section Status.feature

```diff
@@ -201,7 +201,7 @@ Scenario: the order section status is set for an incomplete order with one solut
         | 10001   | 1000-001-A01    | ODS2    |
     When the user makes a request to retrieve the order summary with the ID 10001
     Then a response with status code 200 is returned
-    And the order Section Status is incomplete
+    And the order Section Status is complete
```

Comment:
Similar to above this scenario is incorrect now. When there is a catalogue solution and an additional service in the order, the order section status should only be complete once the associated services section has also been viewed. Therefore this change will need to be reverted.

Reply:
Revert is done